src/views/Rules/ListRules/index.jsx

```diff
@@ -347,14 +356,145 @@ function ListRules(props) {
       throw error;
     }
 
-    if (Object.keys(dialogRule.requiredSignoffs).length > 0) {
+    if (Object.keys(dialogRule.required_signoffs).length > 0) {
       return (await getScheduledChangeByRuleId(dialogRule.rule_id)).data
         .scheduled_changes[0];
     }
 
     return dialogRule.rule_id;
   };
 
+  const signoffDialogBody = (
+    <FormControl component="fieldset">
+      <RadioGroup
+        aria-label="Role"
+        name="role"
+        value={signoffRole}
+        onChange={handleSignoffRoleChange}>
+        {roles.map(r => (
+          <FormControlLabel key={r} value={r} label={r} control={<Radio />} />
+        ))}
+      </RadioGroup>
```

Comment:
Signing off a card makes all the cards show "Revoke Signoff". Is this expected?

![Screen Shot 2019-07-30 at 9 17 24 AM](https://user-images.githubusercontent.com/3766511/62132385-f9676c80-b2aa-11e9-9ea9-1a40ea77835d.png)

Reply:
Definitely not!